OPAL/br/src/main/resources/reference.conf

```diff
@@ -68,7 +68,6 @@ org.opalj {
             {declaringClass = "java/lang/Thread", name = "<init>", descriptor = "(Ljava/lang/ThreadGroup;Ljava/lang/String;)V"},
             {declaringClass = "java/lang/ThreadGroup", name = "<init>", descriptor = "()V"},
             {declaringClass = "java/lang/Thread", name = "exit", descriptor = "()V"},
-            {declaringClass = "java/lang/ThreadGroup+", name = "uncaughtException", descriptor = "(Ljava/lang/Thread;Ljava/lang/Throwable;)V"},
```

Comment:
Can we really remove it completely? This is for the standard uncaughtException. I think only the `+` for overrides should be removed